### PR TITLE
Ampeg Manual Test Project

### PR DESCRIPTION
**Pull request by tchung7**

### CODE REVIEW

File: Manual Testing/Ampeg Manual Test Project.txt [hunk 1]
```diff
@@ -0,0 +1,68 @@
+Test Case #1 
+Create Account with Valid Information
```
Comment (dentaylor) [inline Manual Testing/Ampeg Manual Test Project.txt]: Is this the title?

File: Manual Testing/Ampeg Manual Test Project.txt [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+Entering valid information in create account page
```
Comment (dentaylor) [inline Manual Testing/Ampeg Manual Test Project.txt]: Is this the summary?

File: Manual Testing/Ampeg Manual Test Project.txt [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+Pre conditions -Email, Username, First Name, Last Name, Password, Reenter Password, Country
```
Comment (dentaylor) [inline Manual Testing/Ampeg Manual Test Project.txt]: Not sure what these fields mean. Do I need these values before running the test?
A Pre-condition of this test is that the user account does not already exist.

File: Manual Testing/Ampeg Manual Test Project.txt [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+Traceability to Requirement - Create Account with Application
```
Comment (dentaylor) [inline Manual Testing/Ampeg Manual Test Project.txt:5]: Typically this is a link to bug/feature/etc. We'll cover this more later.

File: Manual Testing/Ampeg Manual Test Project.txt [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+Step 2 - Click "Create Account" button
+Step 3 - Enter Valid Information
+Step 4 - Click "Create your account" button
+Expected Result - Account is successfully created and 
+Account Home Page is loaded.
+
+Test Case #2
+Create Account with Invalid Information
+Enter invalid username in create account page
+Pre conditions -Email, Username, First Name, Last Name, Password, Reenter Password, Country
```
Comment (dentaylor) [inline Manual Testing/Ampeg Manual Test Project.txt]: Same.
Pre-condition is that the user account already exists.

**Review by dentaylor — DISMISSED** [review]
This branch should have been created from develop. Only the manual test txt file should be changed here. DB stuff should not be in here.

File: Manual Testing/Ampeg Manual Test Project.txt (hunk 2 above)
Comment (tchung7) [reply Manual Testing/Ampeg Manual Test Project.txt]: Yes, the third line is the summary

File: Manual Testing/Ampeg Manual Test Project.txt (hunk 1 above)
Comment (tchung7) [reply Manual Testing/Ampeg Manual Test Project.txt]: 2nd line is title

File: Manual Testing/Ampeg Manual Test Project.txt (hunk 3 above)
Comment (tchung7) [reply Manual Testing/Ampeg Manual Test Project.txt]: Got it, will change in the txt file before merge

File: Manual Testing/Ampeg Manual Test Project.txt (hunk 5 above)
Comment (tchung7) [reply Manual Testing/Ampeg Manual Test Project.txt]: Same thing, will change before merge

**Review by dentaylor — APPROVED** [review]

**Pull request merged**